app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+
+
+class Elf(Player):
```

Comment:
You should inherit the `Elf` class from `ABC` as well because it does not implement all abstract methods from the `Player` class.